solver/scheduler_test.go

```diff
@@ -716,7 +716,7 @@ func TestHugeGraph(t *testing.T) {
 	t.Parallel()
 	ctx := context.TODO()
 
-	rand.Seed(time.Now().UnixNano())
+	rand.New(rand.NewSource(time.Now().UnixNano())) //nolint:gosec
```

Comment:
:eyes: I'm confused by this. Doesn't this just [create a new random generator and then immediately discard it](https://pkg.go.dev/math/rand#New)?

Reply:
Yes it seems [Go 1.20 now automatically seeds the global random number generator (used by top-level functions like Float64 and Int) with a random value](https://tip.golang.org/doc/go1.20). There is no way to get the automatically generated seed so I guess we could just remove this. Will take a closer look.